[2023-02-13T14:28:50.789+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: first_dag_v3.task_3 manual__2023-02-13T14:28:48.751035+00:00 [queued]>
[2023-02-13T14:28:50.797+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: first_dag_v3.task_3 manual__2023-02-13T14:28:48.751035+00:00 [queued]>
[2023-02-13T14:28:50.798+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-13T14:28:50.798+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-02-13T14:28:50.799+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-13T14:28:50.811+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): task_3> on 2023-02-13 14:28:48.751035+00:00
[2023-02-13T14:28:50.817+0000] {standard_task_runner.py:55} INFO - Started process 3539 to run task
[2023-02-13T14:28:50.822+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'first_dag_v3', 'task_3', 'manual__2023-02-13T14:28:48.751035+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/dag-1.py', '--cfg-path', '/tmp/tmp3psshfwd']
[2023-02-13T14:28:50.825+0000] {standard_task_runner.py:83} INFO - Job 7: Subtask task_3
[2023-02-13T14:28:50.913+0000] {task_command.py:388} INFO - Running <TaskInstance: first_dag_v3.task_3 manual__2023-02-13T14:28:48.751035+00:00 [running]> on host 38744fc0668d
[2023-02-13T14:28:50.969+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tlysenko
AIRFLOW_CTX_DAG_ID=first_dag_v3
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2023-02-13T14:28:48.751035+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-13T14:28:48.751035+00:00
[2023-02-13T14:28:50.970+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-13T14:28:50.971+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "It is the third task that f"']
[2023-02-13T14:28:50.981+0000] {subprocess.py:86} INFO - Output:
[2023-02-13T14:28:50.984+0000] {subprocess.py:93} INFO - It is the third task that f
[2023-02-13T14:28:50.985+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-13T14:28:51.013+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=first_dag_v3, task_id=task_3, execution_date=20230213T142848, start_date=20230213T142850, end_date=20230213T142851
[2023-02-13T14:28:51.035+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-13T14:28:51.048+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
